src/addTodoForm.js

```diff
@@ -1,18 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
 
-function AddTodoForm(props) {
+function AddTodoForm({onAddTodo}) {
+
+    const [todoTitle, setTodoTitle] = useState('')
+
+    function handleTitleChange(event) {
+        const newTodoTitle = event.target.value
+        setTodoTitle(newTodoTitle)
+    }
 
     function handleAddTodo(event) {
         event.preventDefault()
-        const todoTitle = event.target.title.value
-        props.onAddTodo(todoTitle)
-        event.target.reset()
+        onAddTodo({ title: todoTitle, id: Date.now() })
+        event.target.title.value = ""
```

Comment:
Requires changes:
Since this value is now controlled by state, you'll want to use the `setTodoTitle("")` state setter instead of the `event.target.title` to reset the input's value.

>  - [ ] Inside handleAddTodo, remove the reset() method and replace it with logic to reset the todoTitle state to an empty String